component/main.jsonnet

```diff
@@ -67,14 +67,48 @@ local identityProviders = [
   if idp.type == 'LDAP'
 ];
 
+local ldapSync =
+  local ldapSyncServiceAccount = com.namespaced(params.namespace, kube.ServiceAccount('ldap-sync'));
+
+  [
+    ldapSyncServiceAccount,
+    kube.ClusterRoleBinding('ldap-sync') {
+      subjects_: [{
```

Comment:
You can directly pass the SA object here:

```
subjects_: [ldapSyncServiceAccount]
```

kube.libsonnet will extract the required fields from the ServiceAccount object.